
package com.laegler.openbanking.soap.service;

import javax.xml.ws.WebFault;


/**
 * Bad request
 *
 * This class was generated by Apache CXF 3.3.3
 * 2019-10-21T07:01:26.686+13:00
 * Generated source version: 3.3.3
 */

@WebFault(name = "POST_CreateacallbackURL_400", targetNamespace = "http://laegler.com/openbanking/soap/model")
public class POSTCreateacallbackURL400 extends Exception {

    private com.laegler.openbanking.soap.model.OBErrorResponse1 postCreateacallbackURL400;

    public POSTCreateacallbackURL400() {
        super();
    }

    public POSTCreateacallbackURL400(String message) {
        super(message);
    }

    public POSTCreateacallbackURL400(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public POSTCreateacallbackURL400(String message, com.laegler.openbanking.soap.model.OBErrorResponse1 postCreateacallbackURL400) {
        super(message);
        this.postCreateacallbackURL400 = postCreateacallbackURL400;
    }

    public POSTCreateacallbackURL400(String message, com.laegler.openbanking.soap.model.OBErrorResponse1 postCreateacallbackURL400, java.lang.Throwable cause) {
        super(message, cause);
        this.postCreateacallbackURL400 = postCreateacallbackURL400;
    }

    public com.laegler.openbanking.soap.model.OBErrorResponse1 getFaultInfo() {
        return this.postCreateacallbackURL400;
    }
}
